crates/bitwarden-vault/src/cipher/cipher_client.rs

```diff
@@ -1,15 +1,27 @@
 use bitwarden_core::Client;
 use bitwarden_crypto::IdentifyKey;
-use uuid::Uuid;
+#[cfg(feature = "wasm")]
+use wasm_bindgen::prelude::*;
 
 use crate::{
     Cipher, CipherError, CipherListView, CipherView, DecryptError, EncryptError, VaultClient,
 };
 
+/// NewType representing an Organization ID. This ensures type safety.
+#[cfg_attr(
+    feature = "wasm",
+    derive(serde::Serialize, serde::Deserialize, tsify_next::Tsify),
+    tsify(into_wasm_abi, from_wasm_abi)
+)]
+#[repr(transparent)]
+pub struct OrganizationId(#[tsify(type = r#"Tagged<Uuid, "OrganizationId">"#)] pub uuid::Uuid);
```

Comment:
suggestion: instead of making the internal field public, how about implementing `TryFrom` to convert between types?

Reply:
I think so, I wanted to hold off until I had implemented a macro to avoid writing throw away code but that's done now.